router/router.go

```diff
@@ -312,6 +317,16 @@ func (r *router) AddRealm(config *RealmConfig) error {
 	return err
 }
 
+// GetRouterFeatures exposes WAMP features current version provides.
+func (r *router) GetRouterFeatures() *wamp.Dict {
```

Comment:
I think this should just be called `RouterFeatures`

Reply:
Yeah, renamed to be more go idiomatic